Doctrine/PHPCR/ElasticaToDocumentTransformer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace FOS\ElasticaBundle\Doctrine\PHPCR;
+
+use FOS\ElasticaBundle\Doctrine\AbstractElasticaToModelTransformer;
+
+/**
+ * Maps Elastica documents with Doctrine objects
+ * This mapper assumes an exact match between
+ * elastica documents ids and doctrine object ids
+ */
+class ElasticaToDocumentTransformer extends AbstractElasticaToModelTransformer
+{
+    /**
+     * Fetch objects for theses identifier values
+     *
+     * @param array $identifierValues ids values
+     * @param Boolean $hydrate whether or not to hydrate the objects, false returns arrays
+     * @return array of objects or arrays
+     */
+    protected function findByIdentifiers(array $identifierValues, $hydrate)
+    {
+        // Special case where model is interface. But maybe remove because negative performance impact
+        $reflectionClass = new \ReflectionClass($this->objectClass);
+        if ($reflectionClass->isInterface()) {
```

Comment:
Maybe I'm wrong, but theses methods doesn't seems fit the needs... oO

If I want avoid this Reflection use, but allow such ElasticableInterface use for multiple models in foselastica settings, I could do differently however.
Check in a higher level, during foselastica init configuration, if the model value is an interface case and pass this information to the ElasticaToModelTransformer by example.
But see if its has plus-value.
Or I can also remove this special case and do it only for my project.